java/ql/src/Security/CWE/CWE-200/AndroidWebViewSettingsFileAccess.qhelp

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE qhelp PUBLIC
+ "-//Semmle//qhelp//EN"
+ "qhelp.dtd">
+<qhelp>
+  <overview>
+    <p>
+      File access in an Android WebView can expose the device's file system to
+      the JavaScript running in the WebView. If there are vulnerabilities in the
+      JavaScript or untrusted content is loaded in the WebView, file access may
+      allow an attacker to access or steal the user's data.
+    </p>
+  </overview>
+
+  <recommendation>
+    <p>When possible, you should not allow file access. The file access settings
+    are disabled by default. You can explicitly disbale them by setting the
+    following settings to <code>false</code>:</p>
+
+    <ul>
+      <li><code>setAllowFileAccess</code></li>
+      <li><code>setAllowFileAccessFromFileURLs</code></li>
+      <li><code>setAllowUniversalAccessFromFileURLs</code></li>
+    </ul>
+
```

Comment:
There's some weird spacing/indentation here in the [preview](https://github.com/github/codeql/pull/11241#issuecomment-1312502915) generated by the bot and I'm not sure why 🤔 will that go away in production? Or does that somehow need to be addressed here?

Reply:
I believe this is an issue from the way the Markdown is being rendered. But I will ask others if this is a common issue.

Reply:
I just had a look at other Qhelp files. It seems as if this is a bug in the Markdown generator that we haven't noticed before. (🎖️ for Sam)

If you look at: https://github.com/github/codeql/blob/1bf82cecfa7b7f8365c58d08583985ce84df78e4/python/ql/src/Security/CWE-327/InsecureProtocol.qhelp#L46, it seems as if this should be a new, unindented paragraph. But it's currently being generated as wrapped into the bullet above: https://codeql.github.com/codeql-query-help/python/py-insecure-default-protocol/.